### DISCUSSION TITLE
Traceable Mod Customization

### DISCUSSION THREAD

**Discussion by SealsarecuteR** [ppy/osu#18047] [Ideas] [answered]
I really like the Traceable mod, but I don't like the fact that there are no visible hit circles. I like how the mod looks, but I can't find a skin like it that has hit circles in it, so I think there should be a setting in the customization tab that allows you to choose if the hit circles are visible.

**Comment by peppy** [comment, marked as answer]
No hit circles is part of the mod, though.

Mods are not intended to be used like skins. I'd recommend searching harder for a skin, or waiting until the full skinning system comes online and makes it easier to adjust visuals like this yourself.
